nextflow.config

```diff
@@ -19,6 +19,12 @@ params {
     igenomes_base              = 's3://ngi-igenomes/igenomes/'
     igenomes_ignore            = false
 
+    // Kraken2 options
+    kraken2_db                 = 'https://github.com/nf-core/test-datasets/raw/taxprofiler/data/database/kraken2/testdb-kraken2.tar.gz'
```

Comment:
which db is? there are different ones: https://benlangmead.github.io/aws-indexes/k2 which requires different resources depending on the size. The one used, even if its for test should be documented somewhere (maybe it's and I haven't seen it).

Reply:
I used the minimalest possible database for testing purposes, but I agree with you that it should not be default one, it should just be set to `null`. I used the `taxprofiler` test one, which was built like this: https://github.com/nf-core/test-datasets/blob/taxprofiler/README.md#kraken2